device/celestica/x86_64-cel_e1031-r0/sonic_platform/chassis.py

```diff
@@ -17,6 +17,8 @@
 try:
     from sonic_platform_base.chassis_base import ChassisBase
     from sonic_platform.fan import Fan
+    from sonic_platform.psu import Psu
+    from sonic_platform.module import Module
```

Comment:
ModuleBase and ChassisBase have a number of  common methods. Both ChassisBase and ModuleBase should have firmware-related methods. You should not need to import both.